src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -805,6 +805,42 @@ HTTPS can be enabled for Web applications using a Java keystore. See `web.https`
 for the possible settings. We provided a default self-signed keystore than can be used for testing but for
 production you should build your own keystore with a valid certificate so it will be accepted by browsers.
 
+[[_script_engines]]
+== Configure script engines
+
+Most script engines do not need any configuration. This section talks about the script engines which can be configured
+individually.
+
+=== Docker-compose script engine
```

Comment:
Is it needed to repeat "script engine" since the subsection is part of "Configure script engines"?